Styles/styles.css

```diff
@@ -0,0 +1,111 @@
+h1 {
+    position: absolute;
+    width: 306px;
+    height: 47px;
+    left: 600px;
+    top: 120px;
+}
+hr {
+    position: absolute;
+    width: 100px;
+    height: 4px;
+    left: 670px;
+    top: 187px;
+    background-color: #F5BD41;
+}
+div {
+    display: inline-block;
+}
+.container {
+    position: absolute;
+    width: 296px;
+    height: 183px;
+}
+
+.support {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 202px;
+    top: 271px;
+}
+
+.communication {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 513px;
+    top: 271px;
+}
+.booking {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 826px;
+    top: 271px;
+}
+.reviews {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    left: 1138px;
+    top: 271px;
+}
+.text1 {
```

Comment:
It is not the best way to use number in classname here. There is no clear logic in such numeration. The order of elements could be changed and even the number of elements could be changed.